cpp/src/parquet/arrow/arrow_schema_test.cc

```diff
@@ -1846,7 +1953,9 @@ TEST_F(TestLevels, ListErrors) {
   {
     ::arrow::Status error = MaybeSetParquetSchema(GroupNode::Make(
         "child_list", Repetition::REPEATED,
-        {PrimitiveNode::Make("bool", Repetition::REPEATED, ParquetType::BOOLEAN)},
+        {GroupNode::Make("list", Repetition::REPEATED,
```

Comment:
Original case is a two-level list, which I switched to a three-level list because two-level list should allow repeated.